--- a/include/dyndns.h	2012-09-19 17:12:44.102062600 +0400
+++ b/include/dyndns.h	2012-09-19 17:13:42.858052020 +0400
@@ -51,8 +51,8 @@
 
 /*test values*/
 #define DYNDNS_DEFAULT_DEBUG_LEVEL	1
-#define DYNDNS_DEFAULT_CONFIG_FILE	"/etc/inadyn.conf"
-#define DYNDNS_RUNTIME_DATA_DIR		"/var/run/inadyn"
+#define DYNDNS_DEFAULT_CONFIG_FILE	"/opt/etc/inadyn.conf"
+#define DYNDNS_RUNTIME_DATA_DIR		"/opt/var/run/inadyn"
 #define DYNDNS_DEFAULT_CACHE_FILE	DYNDNS_RUNTIME_DATA_DIR"/inadyn.cache"
 #define DYNDNS_DEFAULT_PIDFILE		DYNDNS_RUNTIME_DATA_DIR"/inadyn.pid"
 
