--- a/rutorrent/conf/config.php
+++ b/rutorrent/conf/config.php
@@ -31,7 +31,7 @@
 $do_diagnostic = true;
 
 // Path to log file. Set to '' to disable logging.
-$log_file = '/tmp/errors.log';
+$log_file = '/opt/tmp/errors.log';
 
 // If set to true, all .torrent files added via webui will be saved in share/torrents
 // (or share/users/USERNAME/torrents) directory. If set to false, all .torrent files
@@ -51,25 +51,25 @@
 
 // '$scgi_port' and '$scgi_host' must match 'scgi_port' setting in rTorrent
 // configuration file.
-$scgi_port = 5000;
-$scgi_host = "127.0.0.1";
+//$scgi_port = 5000;
+//$scgi_host = "127.0.0.1";
 
 // For webserver->rTorrent connection via unix domain socket, change '$scgi_port'
 // and '$scgi_host' variables to something like this ('$scgi_host' must match
 // 'scgi_local' setting in rTorrent configuration file):
-//$scgi_port = 0;
-//$scgi_host = "unix:///tmp/rpc.socket";
+$scgi_port = 0;
+$scgi_host = "unix:///opt/var/rpc.socket";
 
 // DO NOT DELETE THIS LINE!!! DO NOT COMMENT THIS LINE!!!
 $XMLRPCMountPoint = "/RPC2";
 
 // Path to external binaries (e.g. /usr/bin/php). Set to '' to search in $PATH.
 $pathToExternals = array(
-    "php"	=> '',
-    "curl"	=> '',
-    "gzip"	=> '',
-    "id"	=> '',
-    "stat"	=> '',
+    "php"	=> '/opt/bin/php-cgi',
+    "curl"	=> '/opt/bin/curl',
+    "gzip"	=> '/opt/bin/gzip',
+    "id"	=> '/opt/bin/id',
+    "stat"	=> '/opt/bin/stat',
 );
 
 // List of local interfaces.
