--- a/makefile
+++ b/makefile
@@ -17,12 +17,12 @@
 #*******************************************************************************
 # Release flags
 #*******************************************************************************
-CXXFLAGS = -O -g -Wall -Wextra
+#CXXFLAGS = -O -g -Wall -Wextra
 
 #*******************************************************************************
 # Include
 #*******************************************************************************
-INCLUDE = -Itinyxml -Iskein/Optimized_32bit -I/usr/include -I/usr/include/lua5.1
+INCLUDE = -Itinyxml  -Iskein/Optimized_32bit $(CXXFLAGS)
 
 #*******************************************************************************
 # Files to compile and link
@@ -49,7 +49,7 @@
 PtokaX: $(OBJS)
 	cd skein; $(MAKE)
 	cd tinyxml; $(MAKE)
-	$(CXX) $(OBJS) $(CURDIR)/tinyxml/tinyxml.a $(CURDIR)/skein/skein.a -o PtokaX -lstdc++ -lpthread -llua5.1 -lrt -lz
+	$(CXX) $(LDFLAGS) $(OBJS) $(CURDIR)/tinyxml/tinyxml.a $(CURDIR)/skein/skein.a -o PtokaX -lstdc++ -lpthread -llua -lrt -lz
 
 lua52: INCLUDE = -Itinyxml -Iskein/Optimized_32bit -I/usr/include -I/usr/include/lua5.2
 lua52: $(OBJS)
