--- a/config.m4	2012-08-07 16:55:45.000000000 +0400
+++ b/config.m4	2012-08-07 16:55:46.112977146 +0400
@@ -189,24 +189,8 @@
 
 dnl determine the best type
   AC_MSG_CHECKING(for best shared memory type)
-  if test "$mm_shm_ipc" = "yes"; then
     AC_DEFINE(MM_SHM_IPC, 1, [Define if you like to use sysvipc based shared memory])
     msg="sysvipc"
-  elif test "$mm_shm_mmap_anon" = "yes"; then
-    AC_DEFINE(MM_SHM_MMAP_ANON, 1, [Define if you like to use anonymous mmap based shared memory])
-    msg="anonymous mmap"
-  elif test "$mm_shm_mmap_zero" = "yes"; then
-    AC_DEFINE(MM_SHM_MMAP_ZERO, 1, [Define if you like to use mmap on /dev/zero based shared memory])
-    msg="mmap on /dev/zero"
-  elif test "$mm_shm_mmap_posix" = "yes"; then
-    AC_DEFINE(MM_SHM_MMAP_POSIX, 1, [Define if you like to use posix mmap based shared memory])
-    msg="posix mmap"
-  elif test "$mm_shm_mmap_file" = "yes"; then
-    AC_DEFINE(MM_SHM_MMAP_FILE, 1, [Define if you like to use mmap on temporary file shared memory])
-    msg="mmap"
-  else
-    msg="no"
-  fi
   AC_MSG_RESULT([$msg])
   if test "$msg" = "no" ; then
     AC_MSG_ERROR([eaccelerator couldn't detect the shared memory type])
@@ -281,31 +265,8 @@
 
 dnl Determine the best type
   AC_MSG_CHECKING(for best semaphores type)
-  if test "$mm_sem_spinlock" = "yes"; then
-    AC_DEFINE(MM_SEM_SPINLOCK, 1, [Define if you like to use spinlock based semaphores])
-    msg="spinlock"
-  elif test "$mm_sem_ipc" = "yes"; then
-    if test $ea_userid = 0; then
-        AC_MSG_ERROR("You need to pass the user id eaccelerator will be running under when using sysvipc semaphores")
-    else
         AC_DEFINE(MM_SEM_IPC, 1, [Define if you like to use sysvipc based semaphores])
         msg="sysvipc"
-    fi
-  elif test "$mm_sem_fcntl" = "yes"; then
-    AC_DEFINE(MM_SEM_FCNTL, 1, [Define if you like to use fcntl based semaphores])
-    msg="fcntl"
-  elif test "$mm_sem_flock" = "yes"; then
-    AC_DEFINE(MM_SEM_FLOCK, 1, [Define if you like to use flock based semaphores])
-    msg="flock"
-  elif test "$mm_sem_pthread" = "yes"; then
-    AC_DEFINE(MM_SEM_PTHREAD, 1, [Define if you like to use pthread based semaphores])
-    msg="pthread"
-  elif test "$mm_sem_posix" = "yes"; then
-    AC_DEFINE(MM_SEM_POSIX, 1, [Define if you like to use posix based semaphores])
-    msg="posix"
-  else
-    msg="no"
-  fi
   AC_MSG_RESULT([$msg])
   if test "$msg" = "no" ; then
     AC_MSG_ERROR([eaccelerator cannot determine semaphore type, which is required])
