--- a/Makefile	2012-08-14 12:41:25.000000000 +0400
+++ b/Makefile	2012-09-26 16:49:49.164827412 +0400
@@ -1,9 +1,11 @@
 LUA     = lua-5.1.4
 #LUA     = lua-5.2.0
-CFLAGS  = -fno-exceptions -fno-rtti -O2 -I$(LUA) -L$(LUA)
+CFLAGS  += -fno-exceptions -fno-rtti
 SRC     = main.cpp soap.cpp mem.cpp mcast.cpp luaxlib.cpp luaxcore.cpp luajson.cpp luajson_parser.cpp
 STAGING_DIR = /home/shocker/staff/openwrt/staging_dir
 
+all: embedded
+
 x86:
 	make -C $(LUA) CC=gcc a
 	gcc -O2 -c -o md5.o md5c.c
@@ -45,10 +47,8 @@
 embedded:
 	export PATH
 	export STAGING_DIR
-	make -C $(LUA) CC=$(SDK)/gcc a
-	$(SDK)/gcc -O2 -c -o md5.o md5c.c
-	$(SDK)/gcc $(CFLAGS) -DWITH_URANDOM -o xupnpd-$(TARGET) $(SRC) md5.o -lm -llua -ldl
-	$(SDK)/strip xupnpd-$(TARGET)
+	$(CC) -O2 -c -o md5.o md5c.c
+	$(CC) $(CFLAGS) -DWITH_URANDOM $(LDFLAGS) -o xupnpd $(SRC) md5.o -lm -llua -ldl
 
 clean:
 	make -C $(LUA) clean
