--- a/source/Makefile
+++ b/source/Makefile
@@ -101,7 +101,7 @@
 #
 
 
-CFOTHERS := -pipe $(EXTERNAL_FLAGS)
+CFOTHERS := $(EXTERNAL_FLAGS)
 CFWARN := -Wall -Wformat-security
 CFWARN_L := -Wundef
 
@@ -110,7 +110,7 @@
 ifdef ANDROID
 LDFLAGS := $(EXTERNAL_LDFLAGS) $(LDFLAGS)
 else
-LDFLAGS := $(EXTERNAL_LDFLAGS)
+LDFLAGS += $(EXTERNAL_LDFLAGS)
 endif
 
 #
@@ -183,6 +183,7 @@
 	endif
 
 endif
+BUILD_LUA = yes
 ifneq (,$(findstring MINGW,$(uname_S)))
 	GAME = crawl.exe
 	bin_prefix = .
@@ -250,7 +251,7 @@
 LIBPNG := contrib/install/$(ARCH)/lib/libpng.a
 LIBSDLIMAGE := contrib/install/$(ARCH)/lib/libSDL_image.a
 LIBFREETYPE := contrib/install/$(ARCH)/lib/libfreetype.a
-LIBSQLITE := contrib/install/$(ARCH)/lib/libsqlite3.a
+LIBSQLITE := -lsqlite3
 ifdef USE_LUAJIT
 LIBLUA := contrib/install/$(ARCH)/lib/libluajit.a
 else
@@ -259,7 +260,7 @@
 LIBZ := contrib/install/$(ARCH)/lib/libz.a
 
 ifndef CROSSHOST
-	SQLITE_INCLUDE_DIR := /usr/include
+	SQLITE_INCLUDE_DIR := .
 else
 	# This is totally wrong, works only with some old-style setups, and
 	# on some architectures of Debian/new FHS multiarch -- excluding, for
@@ -374,7 +375,6 @@
 
 
 INCLUDES_L += -isystem contrib/install/$(ARCH)/include
-LIBS += -Lcontrib/install/$(ARCH)/lib
 
 INCLUDES_L += -Iutil -I.
 
@@ -700,7 +700,6 @@
 
 # On clang, unknown -Wfoo is merely a warning, thus -Werror.
 CFWARN_L += $(shell w=-Wno-array-bounds;echo|$(GXX) -E -x c++ - -Werror $$w >/dev/null 2>&1 && echo $$w)
-CFWARN_L += $(shell w=-Wno-format-zero-length;echo|$(GXX) -E -x c++ - -Werror $$w >/dev/null 2>&1 && echo $$w)
 CFWARN   += $(shell w=-Wmissing-declarations;echo|$(GXX) -E -x c++ - -Werror $$w >/dev/null 2>&1 && echo $$w)
 CFWARN   += $(shell w=-Wredundant-decls;echo|$(GXX) -E -x c++ - -Werror $$w >/dev/null 2>&1 && echo $$w)
 
@@ -943,10 +942,9 @@
 NC_INCLUDE = $(NC_PREFIX)/include/ncursesw
 endif
 
-INCLUDES_L += -isystem $(NC_INCLUDE)
 
 ifndef TILES
-LIBS += -L$(NC_PREFIX)/lib -l$(NC_LIB)
+LIBS += -l$(NC_LIB)
 endif
 
 endif
@@ -975,7 +973,6 @@
   LDFLAGS += -fuse-ld=gold
 endif
 
-LDFLAGS += $(CFOPTIMIZE) $(CFOPTIMIZE_L)
 
 ifdef REPORT
 CFOTHERS += -ftime-report
@@ -984,7 +981,7 @@
 ifdef ANDROID
 CFLAGS   := $(CFOPTIMIZE) $(CFOTHERS) $(CFWARN) $(CFLAGS)
 else
-CFLAGS   := $(CFOPTIMIZE) $(CFOTHERS) $(CFWARN)
+CFLAGS   += $(CFOTHERS) $(CFWARN)
 endif
 CFLAGS_L := $(CFOPTIMIZE_L) $(DEFINES_L) $(CFWARN_L) $(INCLUDES_L) $(CFOTHERS_L)
 ALL_CFLAGS := $(CFLAGS) $(CFLAGS_L)
@@ -1081,8 +1078,7 @@
 CONTRIB_LIBS += $(LIBLUA)
 endif
 ifdef BUILD_SQLITE
-CONTRIBS += sqlite
-CONTRIB_LIBS += $(LIBSQLITE)
+LIBS += $(LIBSQLITE)
 endif
 
 EXTRA_OBJECTS += version.o
