--- a/source/libutil.h
+++ b/source/libutil.h
@@ -72,7 +72,6 @@
 
 int numcmp(const char *a, const char *b, int limit = 0);
 bool numcmpstr(string a, string b);
-size_t strlcpy(char *dst, const char *src, size_t n);
 
 int strwidth(const char *s);
 int strwidth(const string &s);
--- a/source/libutil.cc
+++ b/source/libutil.cc
@@ -1070,26 +1070,6 @@
 
 mouse_mode mouse_control::ms_current_mode = MOUSE_MODE_NORMAL;
 
-size_t strlcpy(char *dst, const char *src, size_t n)
-{
-    if (!n)
-        return strlen(src);
-
-    const char *s = src;
-
-    while (--n > 0)
-        if (!(*dst++ = *s++))
-            break;
-
-    if (!n)
-    {
-        *dst++ = 0;
-        while (*s++)
-            ;
-    }
-
-    return s - src - 1;
-}
 
 string unwrap_desc(string desc)
 {
