--- a/Makefile.am
+++ b/Makefile.am
@@ -1,30 +1,5 @@
-EXTRA_DIST=ChangeLog doc/ncdc.pod.in
-CLEANFILES=doc/ncdc.1 doc/ncdc.pod
-
-man_MANS=doc/ncdc.1
-
-noinst_PROGRAMS=gendoc
-gendoc_SOURCES=doc/gendoc.c
-
-doc/ncdc.pod: $(srcdir)/doc/ncdc.pod.in gendoc$(EXEEXT)
-	$(AM_V_GEN)./gendoc$(EXEEXT) <"$(srcdir)/doc/ncdc.pod.in" >doc/ncdc.pod
-
-doc/ncdc.1: doc/ncdc.pod Makefile
-if USE_POD2MAN
-	$(AM_V_GEN)pod2man --center "ncdc manual" --release "@PACKAGE@-@VERSION@" doc/ncdc.pod >doc/ncdc.1
-endif
-
-
-if HAVE_MH
-mkhdr=makeheaders
-mkhdr_dep=
-else
 mkhdr=./makeheaders$(EXEEXT)
-mkhdr_dep=makeheaders$(EXEEXT)
-noinst_PROGRAMS+=makeheaders
-endif
-makeheaders_SOURCES=deps/makeheaders.c
-
+mkhdr_dep=
 
 
 bin_PROGRAMS=ncdc
