--- a/GNUmakefile.am	2012-05-03 14:11:19.000000000 +0400
+++ b/GNUmakefile.am	2012-05-04 15:35:58.889706035 +0400
@@ -1,24 +1,3 @@
-EXTRA_DIST=ChangeLog doc/ncdc-db-upgrade.pod doc/ncdc.pod.in
-CLEANFILES=doc/ncdc.1 doc/ncdc-db-upgrade.1 doc/ncdc.pod
-
-man_MANS=doc/ncdc.1
-if ENABLE_DB_UPGRADE
-man_MANS+=doc/ncdc-db-upgrade.1
-endif
-
-noinst_PROGRAMS=gendoc
-gendoc_SOURCES=doc/gendoc.c
-
-doc/ncdc.pod: $(top_srcdir)/doc/ncdc.pod.in gendoc
-	$(AM_V_GEN)./gendoc <$< >$@
-
-${man_MANS}: %.1: %.pod GNUmakefile
-if USE_POD2MAN
-	$(AM_V_GEN)pod2man --center "ncdc manual" --release "@PACKAGE@-@VERSION@" $< >$@
-endif
-
-
-
 bin_PROGRAMS=ncdc
 ncdc_SOURCES=\
 	src/cc.c\
