--- a/configure.in	2012-08-16 16:09:01.209643392 +0400
+++ b/configure.in	2012-08-16 16:11:25.669640096 +0400
@@ -180,15 +180,7 @@
 # Check whether we should use the version string from AC_INIT(), or use
 # git-describe to create one. This trick is copied from the pacman source.
 
-AC_ARG_ENABLE(git-version,
-  AS_HELP_STRING([--enable-git-version], [enable use of git version in version string if available]),
-  [wantgitver=$enableval], [wantgitver=yes])
-
 usegitver=no
-if test "x$wantgitver" = "xyes" ; then
-  AC_CHECK_PROGS([GIT], [git], [no])
-  test "x$GIT" != "xno" -a -d "$srcdir/.git" && usegitver=yes
-fi
 AM_CONDITIONAL(USE_GIT_VERSION, test "x$usegitver" = "xyes")
 
 
