--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,3 @@
-CFLAGS:=-O2 -Wall $(shell pkg-config --cflags openssl ao)
-LDFLAGS:=-lm -lpthread $(shell pkg-config --libs openssl ao)
 OBJS=socketlib.o shairport.o alac.o hairtunes.o
 all: hairtunes shairport
 
@@ -17,7 +15,7 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 
-prefix=/usr/local
+prefix=/opt
 install: hairtunes shairport
 	install -D -m 0755 hairtunes $(DESTDIR)$(prefix)/bin/hairtunes
 	install -D -m 0755 shairport.pl $(DESTDIR)$(prefix)/bin/shairport.pl
