--- a/areastat/Makefile	2012-04-06 23:21:43.000000000 +0400
+++ b/areastat/Makefile	2012-04-06 23:28:27.080188483 +0400
@@ -32,7 +32,7 @@
 SRC_DIR=src/
 
 OBJS= areastat.o
-MANS= man/areastat.1 man/areastat.cfg.5
+MANS= man/areastat.1 man/areastat.conf.5
 
 areastat: $(OBJS)
 		$(CC) $(OBJS) $(LFLAGS) $(LIBS) -o areastat
@@ -78,7 +78,7 @@
 endif
 ifdef MANDIR
 	-$(MKDIR) $(MKDIROPT) $(MANDIR)$(DIRSEP)man1
-	for m in $(MANS); do $(INSTALL) $(IMOPT) $$m.gz $(MANDIR)$(DIRSEP)man`echo $$m | sed s/.*\\.//`/; done
+	for m in $(MANS); do $(INSTALL) $(IMOPT) $$m $(MANDIR)`echo $$m | sed s/.*\\.//`/; done
 endif
 
 uninstall:
