--- a/fidoconf/util/sq2fc.pl	2012-04-07 23:30:27.101793466 +0400
+++ b/fidoconf/util/sq2fc.pl	2012-04-07 23:30:31.673734870 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/opt/bin/perl
 
 # Convert SQUISH.CFG to echoareas's part of fidoconfig
 # REad SQUISH.CFG and SQAFIX.CFG from current directory
--- a/fidoconf/util/fconf2areasbbs.pl	2012-04-07 23:38:01.012086505 +0400
+++ b/fidoconf/util/fconf2areasbbs.pl	2012-04-07 23:38:08.823991159 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/opt/bin/perl
 # $Id: fconf2areasbbs.pl,v 1.2 2011/02/21 23:16:07 stas_degteff Exp $
 ###############################################################################
 # fconf config file to areas.bbs converter
--- a/fidoconf/util/fconf2na.pl	2012-04-07 23:39:23.999043166 +0400
+++ b/fidoconf/util/fconf2na.pl	2012-04-07 23:39:30.022964378 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/opt/bin/perl
 #
 # This script parses HPT echoareas file and produces export areatags and
 # descriptions to FIDONET.NA-compatible file.
--- a/fidoconf/util/fconf2dir.pl	2012-04-07 23:39:03.527302100 +0400
+++ b/fidoconf/util/fconf2dir.pl	2012-04-07 23:39:11.863196304 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/opt/bin/perl
 #
 # $Id: fconf2dir.pl,v 1.1 2003/03/21 08:02:32 sfpavel Exp $
 #
--- a/fidoconf/util/linkedto	2003-03-21 11:02:32.000000000 +0300
+++ b/fidoconf/util/linkedto	2012-04-07 23:47:09.481193133 +0400
@@ -14,10 +14,10 @@
 if [ "$FIDOCONFIG" = "" ] ; then
    if [ -f config ] ; then
       FIDOCONFIG=config
-   elif [ -f /etc/fido/config ]; then
-      FIDOCONFIG="/etc/fido/config"
-   elif [ -f /usr/local/etc/fido/config ]; then
-      FIDOCONFIG="/usr/local/etc/fido/config"
+   elif [ -f /opt/etc/fido/config ]; then
+      FIDOCONFIG="/opt/etc/fido/config"
+   elif [ -f /opt/usr/etc/fido/config ]; then
+      FIDOCONFIG="/opr/usr/etc/fido/config"
    else
       echo "Can't found config. Set the FIDOCONFIG enviroment variable."
    fi
