--- a/mcabberrc.example	2010-12-01 23:51:21.000000000 +0300
+++ b/mcabberrc.example	2012-07-24 02:19:49.000000000 +0400
@@ -1,3 +1,3 @@
-# Sample ~/.mcabber/mcabberrc file
+# Sample /opt/etc/.mcabber/mcabberrc file
 
 # Note about this file syntax:
@@ -100,5 +100,5 @@
 # currently opened buffer. For that set help_to_current option.
 #set lang = en
-#set help_dirs = "/usr/share/mcabber/help;~/share/mcabber/help"
+#set help_dirs = "/opt/share/mcabber/help;~/share/mcabber/help"
 #set help_to_current = 0
 
@@ -142,5 +142,5 @@
 # You can load (read) the messages history: set load_logs = 1
 # If you enable load_logs, you can use the 'max_history_age' setting below.
-# Default logging directory (logging_dir) is $HOME/.mcabber/histo/
+# Default logging directory (logging_dir) is /opt/etc/.mcabber/histo/
 # Defaults for logging, load_logs are 0 (disabled)
 # Note: the logging directory must exist if you enable logging, mcabber
@@ -150,5 +150,5 @@
 #set logging = 1
 #set load_logs = 1
-#set logging_dir = ~/.mcabber/histo/
+#set logging_dir = /opt/etc/.mcabber/histo/
 #set logging_ignore_status = 1
 
@@ -169,5 +169,5 @@
 # so that the message flags are set back at next startup.
 # Note that 'logging' must be enabled for this feature to work.
-#set statefile = ~/.mcabber/mcabber.state
+#set statefile = /opt/etc/.mcabber/mcabber.state
 
 # You can specify a maximum number of data blocks per buffer (1 block contains
@@ -195,5 +195,5 @@
 # to the directory where your modules reside. Though, default compiled-in
 # value should be appropriate.
-#set modules_dir = /usr/lib/mcabber/
+#set modules_dir = /opt/lib/mcabber/
 
 # Beep
@@ -223,10 +223,10 @@
 #   (x=attention y=muc unread buffers z=muc unread buffers with attention sign)
 # See sample script in contrib/ directory.
-#set events_command = ~/.mcabber/eventcmd
+#set events_command = /opt/etc/.mcabber/eventcmd
 #
 #set events_ignore_active_window = 0
 #
 #set event_log_files = 0
-#set event_log_dir = ~/.mcabber/event_files
+#set event_log_dir = /opt/etc/.mcabber/event_files
 
 # If you set 'eventcmd_use_nickname' to 1, mcabber will pass the nickname
@@ -260,5 +260,5 @@
 # the 'fifo_ignore' option (default: 0).  When set to 1, the FIFO input is
 # still read but it is discarded.
-#set fifo_name = ~/.mcabber/mcabber.fifo
+#set fifo_name = /opt/etc/.mcabber/mcabber.fifo
 #set fifo_hide_commands = 0
 #set fifo_ignore = 0
@@ -273,5 +273,5 @@
 # Default is level 0, no trace logging
 #set tracelog_level = 1
-#set tracelog_file = ~/.mcabber/mcabber.log
+#set tracelog_file = /opt/etc/.mcabber/mcabber.log
 
 # Set the auto-away timeout, in seconds.  If set to a value >0,
@@ -475,6 +475,6 @@
 # OTR directory
 # You can specify the OTR directory with the otr_dir option
-# (default = ~/.mcabber/otr/).
-#set otr_dir = "~/.mcabber/otr/"
+# (default = /opt/etc/.mcabber/otr/).
+#set otr_dir = "/opt/etc/.mcabber/otr/"
 #
 # You can set OTR policies from the configuration file (use /help otrpolicy
