--- a/tspc-advanced/template/linux.sh	2012-02-01 11:08:39.166374852 +0400
+++ b/tspc-advanced/template/linux.sh	2012-02-01 11:11:41.188088917 +0400
@@ -74,9 +74,9 @@
 
 ifconfig=/sbin/ifconfig
 route=/sbin/route
-ipconfig=/sbin/ip
+ipconfig=/usr/sbin/ip
 rtadvd=/usr/sbin/radvd
-rtadvd_pid=/var/run/radvd/radvd.pid
+rtadvd_pid=/var/run/radvd.pid
 sysctl=/sbin/sysctl
 rtadvdconfigfilename=gw6c-rtadvd.conf
 rtadvdconfigfile=$TSP_HOME_DIR/$rtadvdconfigfilename
