--- a/micro_proxy.c	2012-07-12 15:33:57.000000000 +0400
+++ b/micro_proxy.c	2012-08-20 11:12:39.419737456 +0400
@@ -38,7 +38,6 @@
 #include <netinet/in.h>
 #include <netdb.h>
 #include <fcntl.h>
-#include <syslog.h>
 
 
 #define SERVER_NAME "micro_proxy"
@@ -81,9 +80,6 @@
     if ( url == (char*) 0 )
 	send_error( 400, "Bad Request", (char*) 0, "Null URL." );
 
-    openlog( "micro_proxy", 0, LOG_DAEMON );
-    syslog( LOG_INFO, "proxying %s", url );
-
     if ( strncasecmp( url, "http://", 7 ) == 0 )
 	{
 	(void) strncpy( url, "http", 4 );	/* make sure it's lower case */
