--- ./source/libutil.h.orig	2013-06-01 06:52:08.000000000 +0400
+++ ./source/libutil.h	2013-06-10 15:37:55.577456987 +0400
@@ -72,7 +72,6 @@
 
 int numcmp(const char *a, const char *b, int limit = 0);
 bool numcmpstr(string a, string b);
-size_t strlcpy(char *dst, const char *src, size_t n);
 
 int strwidth(const char *s);
 int strwidth(const string &s);
--- ./source/libutil.cc.orig	2013-06-01 06:52:08.000000000 +0400
+++ ./source/libutil.cc	2013-06-10 15:38:19.697454826 +0400
@@ -1056,26 +1056,6 @@
 
 mouse_mode mouse_control::ms_current_mode = MOUSE_MODE_NORMAL;
 
-size_t strlcpy(char *dst, const char *src, size_t n)
-{
-    if (!n)
-        return strlen(src);
-
-    const char *s = src;
-
-    while (--n > 0)
-        if (!(*dst++ = *s++))
-            break;
-
-    if (!n)
-    {
-        *dst++ = 0;
-        while (*s++)
-            ;
-    }
-
-    return s - src - 1;
-}
 
 string unwrap_desc(string desc)
 {
