--- a/make-linux.mk
+++ b/make-linux.mk
@@ -1,6 +1,6 @@
 # Pick clang or gcc, with preference for clang
-CC=$(shell which clang gcc cc 2>/dev/null | head -n 1)
-CXX=$(shell which clang++ g++ c++ 2>/dev/null | head -n 1)
+CC?=$(shell which clang gcc cc 2>/dev/null | head -n 1)
+CXX?=$(shell which clang++ g++ c++ 2>/dev/null | head -n 1)
 
 INCLUDES=
 DEFS=
@@ -10,27 +10,6 @@
 OBJS+=osnet/LinuxRoutingTable.o osnet/LinuxEthernetTap.o osnet/LinuxEthernetTapFactory.o
 TESTNET_OBJS=testnet/SimNet.o testnet/SimNetSocketManager.o testnet/TestEthernetTap.o testnet/TestEthernetTapFactory.o testnet/TestRoutingTable.o
 
-# Enable SSE-optimized Salsa20 on x86 and x86_64 machines
-MACHINE=$(shell uname -m)
-ifeq ($(MACHINE),x86_64)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-ifeq ($(MACHINE),amd64)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-ifeq ($(MACHINE),i686)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-ifeq ($(MACHINE),i586)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-ifeq ($(MACHINE),i386)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-ifeq ($(MACHINE),x86)
-	DEFS+=-DZT_SALSA20_SSE 
-endif
-
 # "make official" is a shortcut for this
 ifeq ($(ZT_OFFICIAL_RELEASE),1)
 	ZT_AUTO_UPDATE=1
@@ -50,9 +29,9 @@
 	# C25519 in particular is almost UNUSABLE in heavy testing without it.
 ext/lz4/lz4.o node/Salsa20.o node/SHA512.o node/C25519.o node/Poly1305.o: CFLAGS = -Wall -O2 -g -pthread $(INCLUDES) $(DEFS)
 else
-	CFLAGS=-Wall -O3 -fPIE -fvisibility=hidden -fstack-protector -pthread $(INCLUDES) -DNDEBUG $(DEFS)
-	LDFLAGS=-pie -Wl,-z,relro,-z,now
-	STRIP=strip --strip-all
+	CFLAGS+=-Wall -fPIE -fvisibility=hidden -pthread $(INCLUDES) -DNDEBUG $(DEFS)
+	LDFLAGS+=-pie -Wl,-z,relro,-z,now -lpthread
+	STRIP?=strip --strip-all
 endif
 
 # Uncomment for gprof profile build
@@ -60,7 +39,7 @@
 #LDFLAGS=
 #STRIP=echo
 
-CXXFLAGS=$(CFLAGS) -fno-rtti
+CXXFLAGS+=-fno-rtti -fPIC
 
 all:	one
 
@@ -75,7 +54,7 @@
 	$(STRIP) zerotier-selftest
 
 testnet: $(TESTNET_OBJS) $(OBJS) testnet.o
-	$(CXX) $(CXXFLAGS) -o zerotier-testnet testnet.o $(OBJS) $(TESTNET_OBJS) $(LIBS)
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) -o zerotier-testnet testnet.o $(OBJS) $(TESTNET_OBJS) $(LIBS)
 	$(STRIP) zerotier-testnet
 
 installer: one FORCE
