--- ./CMakeLists.txt.orig	2013-04-12 11:29:37.000000000 +0400
+++ ./CMakeLists.txt	2013-04-12 11:48:44.775006472 +0400
@@ -1,7 +1,11 @@
 cmake_minimum_required(VERSION 2.6)
 
 PROJECT(uqmi C)
-ADD_DEFINITIONS(-Os -ggdb -Wall -Werror --std=gnu99 -Wmissing-declarations)
+ADD_DEFINITIONS(-Wall -Werror --std=gnu99 -Wmissing-declarations)
+
+include_directories (
+    ${OPTIONAL_INCLUDE_DIR}
+    )
 
 SET(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "")
 
