--- a/cloudprint/cloudprint.py
+++ b/cloudprint/cloudprint.py
@@ -33,8 +33,8 @@
         self.printer_id = None
         self.cups= cups.Connection()
         self.proxy =  platform.node() + '-Armooo-PrintProxy'
-        self.auth_path = os.path.expanduser('~/.cloudprintauth')
-        self.xmpp_auth_path = os.path.expanduser('~/.cloudprintauth.sasl')
+        self.auth_path = '/opt/var/.cloudprintauth'
+        self.xmpp_auth_path = '/opt/var/.cloudprintauth.sasl'
         self.username = None
         self.password = None
 
@@ -378,7 +378,7 @@
     print '-d\t\t: enable daemon mode (requires the daemon module)'
     print '-l\t\t: logout of the google account'
     print '-p pid_file\t: path to write the pid to (default cloudprint.pid)'
-    print '-a account_file\t: path to google account ident data (default ~/.cloudprintauth)'
+    print '-a account_file\t: path to google account ident data (default /opt/var/.cloudprintauth)'
     print '\t\t account_file format:\t <Google username>'
     print '\t\t\t\t\t <Google password>'
     print '-h\t\t: display this help'
