--- a/cloudprint/cloudprint.py	2012-09-26 13:55:18.929043017 +0400
+++ b/cloudprint/cloudprint.py	2012-09-26 14:03:39.921032763 +0400
@@ -27,8 +27,8 @@
         self.printer_id = None
         self.cups= cups.Connection()
         self.proxy =  platform.node() + '-Armooo-PrintProxy'
-        self.auth_path = os.path.expanduser('~/.cloudprintauth')
-        self.xmpp_auth_path = os.path.expanduser('~/.cloudprintsaslauth')
+        self.auth_path = '/opt/var/.cloudprintauth'
+        self.xmpp_auth_path = '/opt/var/.cloudprintsaslauth'
 
     def get_auth(self):
         if self.auth:
