--- a/areastat/Makefile
+++ b/areastat/Makefile
@@ -26,7 +26,7 @@
 ifeq ($(SHORTNAME), 1)
   LIBS  = -L$(LIBDIR) -lfidoconf -lsmapi
 else
-  LIBS  = -L$(LIBDIR) -lfidoconfig -lsmapi
+  LIBS  = -L$(LIBDIR) -lfidoconfig -lsmapi -lhusky
 endif
 
 CDEFS=-D$(OSTYPE) -DUNAME=\"$(UNAME)\" $(ADDCDEFS)
@@ -81,7 +81,7 @@
 endif
 ifdef MANDIR
 	-$(MKDIR) $(MKDIROPT) $(MANDIR)$(DIRSEP)man1
-	for m in $(MANS); do $(INSTALL) $(IMOPT) $$m.gz $(MANDIR)$(DIRSEP)man`echo $$m | sed s/.*\\.//`/; done
+	for m in $(MANS); do $(INSTALL) $(IMOPT) $$m $(MANDIR)`echo $$m | sed s/.*\\.//`/; done
 endif
 
 uninstall:
