--- a/huskybse/huskymak.cfg
+++ b/huskybse/huskymak.cfg
@@ -116,7 +116,7 @@
 
 # update dynamic library cache
 # If your operating system does not need this, you may comment out this line.
-LDCONFIG=/sbin/ldconfig
+
 
 # rename a file
 MV=mv
@@ -170,7 +170,7 @@
 # to avoid linkage with old .so files that may be floating around
 # (see 'ifeq ( $(DYNLIBS), 0 )' conditions)
 #
-DYNLIBS=1
+DYNLIBS=0
 
 # C-compiler: specify name of executable
 EXENAMEFLAG=-o
@@ -179,7 +179,7 @@
 WARNFLAGS=-Wall
 
 # C-compiler: optimization
-OPTCFLAGS=-c -s -O3 -fomit-frame-pointer -fstrength-reduce -fPIC
+OPTCFLAGS=-c -s -O2 -fomit-frame-pointer -fstrength-reduce -fPIC
 
 ifeq ( $(DYNLIBS), 0 )
   ifeq ($(OSTYPE), UNIX)
@@ -216,7 +216,7 @@
 DEBLFLAGS=-g
 
 # options for installing programs
-OPTIBOPT=-c -s -m 555
+OPTIBOPT=-c -m 555
 DEBIBOPT=-c -m 555
 
 ifeq ($(DEBUG), 1)
