--- a/whois.conf	2008-07-17 03:02:39.000000000 +0400
+++ b/whois.conf	2012-07-01 00:18:09.000000000 +0400
@@ -9,3 +9,9 @@
 # Eg: 
 # \.nz$ nz.whois-servers.net
 #
+\.com\.ru$ whois.nic.ru
+\.net\.ru$ whois.nic.ru
+\.org\.ru$ whois.nic.ru
+\.pp\.ru$  whois.nic.ru
+\.msk\.ru$ whois.nic.ru
+\.spb\.ru$ whois.nic.ru
--- a/config.h	2010-12-01 03:37:59.000000000 +0300
+++ b/config.h	2012-07-01 00:32:55.000000000 +0400
@@ -12,5 +12,5 @@
 /* Configuration file */
-/*
-#define CONFIG_FILE "/etc/whois.conf"
-*/
+
+#define CONFIG_FILE "/opt/etc/whois.conf"
+
 
