--- ./accel-pppd/ctrl/l2tp/l2tp.c.orig	2014-04-02 16:00:26.000000000 +0400
+++ ./accel-pppd/ctrl/l2tp/l2tp.c	2014-04-02 16:03:02.663220877 +0400
@@ -4315,8 +4315,6 @@
 	fd = socket(AF_PPPOX, SOCK_DGRAM, PX_PROTO_OL2TP);
 	if (fd >= 0)
 		close(fd);
-	else if (system("modprobe -q pppol2tp || modprobe -q l2tp_ppp"))
-		log_warn("unable to load l2tp kernel module\n");
 
 	l2tp_conn = _malloc((UINT16_MAX + 1) * sizeof(struct l2tp_conn_t *));
 	memset(l2tp_conn, 0, (UINT16_MAX + 1) * sizeof(struct l2tp_conn_t *));
--- ./accel-pppd/ctrl/pppoe/pppoe.c.orig	2014-04-02 16:00:26.000000000 +0400
+++ ./accel-pppd/ctrl/pppoe/pppoe.c	2014-04-02 16:03:29.067221798 +0400
@@ -1520,8 +1520,6 @@
 	fd = socket(AF_PPPOX, SOCK_DGRAM, PX_PROTO_OE);
 	if (fd >= 0)
 		close(fd);
-	else if (system("modprobe -q pppoe"))
-		log_warn("failed to load pppoe kernel module\n");
 
 	conn_pool = mempool_create(sizeof(struct pppoe_conn_t));
 	pado_pool = mempool_create(sizeof(struct delayed_pado_t));
--- ./accel-pppd/ctrl/pptp/pptp.c.orig	2014-04-02 16:00:26.000000000 +0400
+++ ./accel-pppd/ctrl/pptp/pptp.c	2014-04-02 16:03:53.031220013 +0400
@@ -776,8 +776,6 @@
 	fd = socket(AF_PPPOX, SOCK_DGRAM, PX_PROTO_PPTP);
 	if (fd >= 0)
 		close(fd);
-	else if (system("modprobe -q pptp"))
-		log_warn("failed to load pptp kernel module\n");
 
 	serv.hnd.fd = socket(PF_INET, SOCK_STREAM, 0);
 	if (serv.hnd.fd < 0) {
