--- ./configure.ac.orig	2012-06-04 10:54:03.000000000 +0400
+++ ./configure.ac	2012-06-14 14:08:01.525902281 +0400
@@ -36,44 +36,10 @@
   AC_DEFINE(CROSS_COMPILING,,[define if you are cross-compiling])
 )
 
-AX_CHECK_COMPILE_FLAG([-fPIE], [
-  AX_CHECK_LINK_FLAG([-fPIE],
-    [CFLAGS="$CFLAGS -fPIE"]
-  )
-])
-
 AX_CHECK_COMPILE_FLAG([-fcatch-undefined-c99-behavior], [CFLAGS="$CFLAGS -fcatch-undefined-c99-behavior"])
 AX_CHECK_COMPILE_FLAG([-fno-strict-aliasing], [CFLAGS="$CFLAGS -fno-strict-aliasing"])
 AX_CHECK_COMPILE_FLAG([-fno-strict-overflow], [CFLAGS="$CFLAGS -fno-strict-overflow"])
 
-AS_IF([echo `(uname -s) 2>/dev/null` | $GREP "CYGWIN" > /dev/null], [
-  AX_CHECK_LINK_FLAG([-Wl,--dynamicbase], [LDFLAGS="$LDFLAGS -Wl,--dynamicbase"])
-  AX_CHECK_LINK_FLAG([-Wl,--nxcompat], [LDFLAGS="$LDFLAGS -Wl,--nxcompat"])
-], [
-  AS_IF([test `(uname -s) 2>/dev/null` = "DragonFly"],
-    [
-      AX_CHECK_COMPILE_FLAG([-fstack-protector], [
-        AX_CHECK_LINK_FLAG([-fstack-protector],
-          [CFLAGS="$CFLAGS -fstack-protector"]
-        )
-      ])
-    ],
-    [
-      AX_CHECK_COMPILE_FLAG([-fstack-protector-all], [
-        AX_CHECK_LINK_FLAG([-fstack-protector-all],
-          [CFLAGS="$CFLAGS -fstack-protector-all"]
-        )
-      ])
-    ]
-  )
-  AS_IF([test "x$ac_compiler_gnu" = "xyes"],[
-    AS_IF([$CC --version 2>/dev/null | $GREP "Free Software Foundation" > /dev/null 2>&1],[
-      CPPFLAGS="$CPPFLAGS -D_FORTIFY_SOURCE=2"
-      AX_CHECK_COMPILE_FLAG([--param ssp-buffer-size=1], [CFLAGS="$CFLAGS --param ssp-buffer-size=1"])
-    ])
-  ])
-])
-
 AX_CHECK_COMPILE_FLAG([-Wbounded], [CFLAGS="$CFLAGS -Wbounded"])
 AX_CHECK_COMPILE_FLAG([-Winit-self], [CFLAGS="$CFLAGS -Winit-self"])
 AX_CHECK_COMPILE_FLAG([-Wwrite-strings], [CFLAGS="$CFLAGS -Wwrite-strings"])
@@ -102,11 +68,6 @@
 AX_CHECK_COMPILE_FLAG([-Wswitch-enum], [CWFLAGS="$CWFLAGS -Wswitch-enum"])
 AX_CHECK_COMPILE_FLAG([-Wvariable-decl], [CWFLAGS="$CWFLAGS -Wvariable-decl"])
 
-AX_CHECK_LINK_FLAG([-pie], [LDFLAGS="$LDFLAGS -pie"])
-AX_CHECK_LINK_FLAG([-Wl,-z,relro], [LDFLAGS="$LDFLAGS -Wl,-z,relro"])
-AX_CHECK_LINK_FLAG([-Wl,-z,now], [LDFLAGS="$LDFLAGS -Wl,-z,now"])
-AX_CHECK_LINK_FLAG([-Wl,-z,noexecstack], [LDFLAGS="$LDFLAGS -Wl,-z,noexecstack"])
-
 AC_ARG_VAR([RONN], [path to the ronn text processor])
 AC_PATH_PROG([RONN], [ronn], [#])
 
@@ -127,14 +88,6 @@
   PROBES_SOURCE="probes_no_dtrace.h"
 ])
 
-AS_IF([test -d /usr/local/include], [
-  CPPFLAGS="$CPPFLAGS -I/usr/local/include"
-])
-
-AS_IF([test -d /usr/local/lib], [
-  LDFLAGS="$LDFLAGS -L/usr/local/lib"
-])
-
 LIBS="$PTHREAD_LIBS $LIBS"
 CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
 CC="$PTHREAD_CC"
@@ -310,9 +263,7 @@
                  src/libnacl/Makefile
                  src/libnacl/tests/Makefile
                  src/libnacl/okcompilers/c
-                 src/libnacl/okcompilers/do
-                 src/libuv/Makefile
-                 test/Makefile])
+                 src/libnacl/okcompilers/do])
 
 AC_OUTPUT
 
