--- a/src/Makefile.am	2012-06-04 10:54:03.000000000 +0400
+++ b/src/Makefile.am	2012-06-14 11:18:50.006170666 +0400
@@ -1,5 +1,4 @@
 
 SUBDIRS = \
 	dnscrypt-proxy \
-	libuv \
     libnacl
--- a/src/dnscrypt-proxy/Makefile.am	2012-06-04 10:54:03.000000000 +0400
+++ b/src/dnscrypt-proxy/Makefile.am	2012-06-14 13:33:04.507755704 +0400
@@ -46,16 +46,11 @@
 AM_CFLAGS = @CWFLAGS@
 
 AM_CPPFLAGS = \
-	-I../libuv/include \
 	-I../@NACL_PATH@/include/local
 
 dnscrypt_proxy_LDADD = \
-	../libuv/uv.a \
 	../@NACL_PATH@/lib/local/libnacl.a
 
-dnscrypt_proxy_DEPENDENCIES = \
-	../libuv/.done
-
 BUILT_SOURCES = \
 	../libnacl/.done \
 	probes.h
@@ -63,9 +58,6 @@
 ../libnacl/.done:
 	cd ../libnacl && $(MAKE) $(AM_MAKEFLAGS)
 
-../libuv/.done:
-	cd ../libuv && $(MAKE) $(AM_MAKEFLAGS)
-
 probes.h: @PROBES_SOURCE@
 	cat @PROBES_SOURCE@ > $@
 
