--- ./accel-pppd/CMakeLists.txt.orig	2013-10-15 11:06:58.653525631 +0400
+++ ./accel-pppd/CMakeLists.txt	2013-10-15 11:07:13.417525847 +0400
@@ -106,11 +106,3 @@
 )
 
 INSTALL(FILES accel-ppp.conf.5 DESTINATION share/man/man5)
-
-IF (NOT DEFINED CPACK_TYPE)
-	INSTALL(FILES accel-ppp.conf DESTINATION ${CMAKE_FIND_ROOT_PATH}/etc RENAME accel-ppp.conf.dist)
-
-	INSTALL(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_FIND_ROOT_PATH}/var/log/accel-ppp)")
-	INSTALL(CODE "EXECUTE_PROCESS(COMMAND mkdir -p ${CMAKE_FIND_ROOT_PATH}/var/lib/accel-ppp)")
-ENDIF (NOT DEFINED CPACK_TYPE)
-
