--- a/driver/if_pppox.h	2012-09-19 16:53:47.502075774 +0400
+++ b/driver/if_pppox.h	2012-09-19 16:54:11.998082443 +0400
@@ -55,7 +55,7 @@
  * Protocols supported by AF_PPPOX
  */
 #define PX_PROTO_OE    0 /* Currently just PPPoE */
-#if LINUX_VERSION_CODE <= KERNEL_VERSION(2,6,22)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,22)
 #define PX_PROTO_PPTP  1
 #define PX_MAX_PROTO   2
 #else
--- a/accel-pppd/include/if_pppox.h	2012-09-19 16:54:35.070081943 +0400
+++ b/accel-pppd/include/if_pppox.h	2012-09-19 16:54:41.810076449 +0400
@@ -51,7 +51,7 @@
  * Protocols supported by AF_PPPOX
  */
 #define PX_PROTO_OE    0 /* Currently just PPPoE */
-#if LINUX_VERSION_CODE <= KERNEL_VERSION(2,6,22)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,22)
 #define PX_PROTO_PPTP  1
 #define PX_MAX_PROTO   2
 #else
